packages/module/src/catalog/QuickStartTileHeader.scss

```diff
@@ -9,4 +9,9 @@
   & .pf-v5-c-badge:not(:last-of-type) {
     margin-right: var(--pf-v5-global--spacer--sm);
   }
+
```

Comment:
I'm so conditioned to avoid writing custom CSS.

So my instinct here is to say that I think you could replace this `div.pfext-quick-start-title-header__display-name` with :
```
<Flex  justifyContent={{ default: 'justifyContentCenter' }}>
       <Title headingLevel="h3" data-test="title" id={quickStartId}>
          <QuickStartMarkdownView content={name} />
        </Title>
        {action}
</Flex>
```
And you might opt to wrap the children in a `FlexItem` but i don't think that'll be necessary.

Reply:
agree. i'm just use to custom css breaking i try to avoid it.

Reply:
i'm still concerned about the action though. since i think you can't just really put anything there and someone may try to do that.